lib/Dialect/Arc/Transforms/FindInitialVectors.cpp

```diff
@@ -130,6 +131,11 @@ struct Vectorizer {
   TopologicalOrder order;
   Block *block;
 };
+
+size_t vecOps = 0;
+size_t savedOps = 0;
+size_t bigSeedVec = 0;
+size_t vecCreated = 0;
```

Comment:
Please don't use global variables; it's not thread-safe since multiple instances of `FindInitialVectorsPass` would access the same variables. You can make them members of the pass class instead (potentially creating a struct that groups them and thus makes it easier to pass them as function arguments).